docs/guides/formulasC2/reference.md

```diff
@@ -0,0 +1,255 @@
+
+## Triggers
+
+Triggers are the actions that kick off a formula. Triggers can be one of the following type:
+
+* [Event](#event)
+* [Element Request](#element-request)
+* [Scheduled](#scheduled)
+* [Manual](#manual)
+
+### Event <img src="img/trigger-icon-event.png" alt="Event" class="inlineImage">
+
+You can set up triggers that listen for an event to happen on an element instance. To set up this trigger, you must use an Element Instance Variable that, when specified in a formula instance, refers to an element instance that is configured to use webhooks or polling to listen for events.
+
+To set up an Event trigger, you must specify an Element Instance Variable. Click <img src="img/btn-Add.png" alt="Alt Text" class="inlineImage"> to find or create a variable to represent the element instance that will kick off a formula instance when an event occurs.
+
+If an Event trigger's Element Instance is set up for polling instead of webhooks, then each object that is found while polling triggers a separate formula execution. For example, if the poller finds five changes, five different formula executions kick off.
+
+To see event triggers in action, see the following examples:
+
+* [CRM to Messages](examples.html)
+* [Retrieve, Transform, and Sync Contact](examples.html#retrieve-transform-and-sync-contact)
+
+{% include Formulas/scope-trigger-event.md %}
+
+### Element Request <img src="img/trigger-icon-erequest.png" alt="Element Request" class="inlineImage">
+
+Triggered any time a specific API call is made to a given Element Instance. To set up this trigger, you must use an Element Instance Variable that, when specified in a formula instance, refers to an element instance.
+
+When you set up an Element Request trigger, specify the following parameters:
+
+* Element Instance Variable: Click <img src="img/btn-Add.png" alt="Alt Text" class="inlineImage"> to find or create a variable to represent the element that will kick off a formula instance when the specified API call occurs.
+* Method:  {{site.data.table-desc.formula-method}}.
+* API: The endpoint, such as `hubs/crm/contacts`.
+
+### Scheduled <img src="img/trigger-icon-scheduled.png" alt="Scheduled" class="inlineImage">
+
+Triggered at times specified by a Cron job. We recommend that you review the many reference pages for Cron jobs online.
+
+{% include note.html content=" The minimum scheduled frequency is 15 minutes." %}
+
+In general, the Cron format consists of:
+
+| Minute </br>0-59| Hour</br>0-23   | Day of Month</br>1-31   | Month of Year</br>1-12   | Day of Week</br>1-7</br>Monday-Sunday   | Year</br>1900-3000   |
+| :------------- | :------------- | :------------- | :------------- | :------------- |:------------- |
+|  \*  |  \*  |  \*  |  \*  |  \*  |  \*  |
+
+#### Examples
+
+* Run each minute
+
+        	0 0/1 * 1/1 * ? *
```

Comment:
Since the minimum is 15 minutes as specified above, lets not show an example of every minute and give them any funny ideas. ⚔️